### ISSUE TITLE
Master Project Planning Process - 4 of 5 Steps Due Pre-F2F 

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#356]
**PROJECT PLAN - The project is defined as launching our R01/IIR Trials.** 

**Summary**

- 4 of 5 Planning steps (see below) are due before we get together F2F)

- Discussed during our "All Hands" RESEARCH meeting - Tuesday 10AM Pacific

- Discussed during our "All hands" DEVELOPMENT meeting - Thursday 8AM Pacific

- Task list and time estimate (in weeks) due to HQ by 5/29 for our 5/31 MTL Launch meeting.

**1. Official project PLAN process underway, in advance of F2F.**

- This will become a master plan we can evaluate our contributions against to make sure we are making progress on time in the right order, across our dependent workgroups, and the necessary research, development and training requirements of R01/IIR Project Launch.

**1st Step - MASTER TASK LIST** - Brainstorm until you have identified every task you think needs to be done before launch. Principally, make sure that we don't miss a single task that's on your own plate, or within your scope of responsibilities. But, thinking in general to make sure no tasks are missed is very helpful too.

- **DUE NOW - We need the 1st Step and 2nd Step DONE by COB Eastern time Wed 5/29** so that it can be translated to 1 post it with time estimate for that task in weeks by Lindsey & Stacey by COB Pacific Time Wed 5/29, for our 2-hour Thursday 5/30AM MTL Launch meeting.

- CLOSE OPEN ACTION ITEMS IN LUCID/GitHub

**2nd Step - ESTIMATION of the Time needed - WEEKS**

- Elapsed, serial weeks until launch

- Typical week, what can realistic be done

- **DUE NEXT WEEK (by Wed ) Discuss your tasks and finalize your estimates in your own workgroups so that it can be translated to 1 post it with time estimate** (i.e., estimted completion time) for that task in weeks by Lindsey & Stacey by COB Pacific Time Wed 5/29, for our 2-hour Thursday 5/30AM MTL Launch meeting.

**3rd Step - Network Diagram - "Critical Path of Dependent Work"** 

- aka PERT 

- e.g., SP algorithm - 2 weeks (Andrew), SAY guides - 2 weeks (largely Debbie & Jane as point, but all hands from Team PSD Facilitators), train TAS - 2 weeks (Facilitate Workgroup - has to happen in this serial order, only some can be done in parallel (there's no way to get in less than weeks)

- **During our 2-hour Thursday 5/30AM MTL Launch meeting, we will use the task estimates to identify the task dependencies or  "Critical path" across each workgroups' task list.** 

1. A critical path -  is the dependent sequence of project task which add up to the longest overall duration. This determines the shortest time possible to our R01/IIR project launch. 

2. We may have more than one critical path due to multiple dependencies between tasks, or separate task sequences that run for the same duration.

**4th Step - GANTT**

**Based on our Critical path, we'll fill in a GANTT to clarify how long it will take to be ready for R01/IIR launch**. The GANTT will help us refine our plan for the order in which you'll complete tasks, with a more realistic assessment of the time across dependent tasks in each week.

- Review the serial dependent paths to launch based on the critical path

- See where there overlap, where tasks can be done in parallel, or tasks that will "nest two birds with one birds"

- Identify/clarify floating tasks (or slack) where

- **DRAFT GANTT from each working group, and our 5/30 meeting - so we have it in the morning Thursday 6/6 at 8AM Pacific**

**5th Step - GANTT of GANTTs**

- For our completed GANTTs, review the workgroup interdependencies across our GANTT charts that will help us monitor our progress up to the R01/IIR launch

- At a minimum likely we need a GANTT for 1) Research Tasks, 2) Development Tasks, 3) Training tasks.

- **FINISH TOGETHER at F2F Thursday 8AM 6/13**

**Comment by dlounsbu** [comment]
[LOUNSBURY_MTL_go_plan_2019-05-28.xlsx](https://github.com/lzim/teampsd/files/3230398/LOUNSBURY_MTL_go_plan_2019-05-28.xlsx)

@lzim Here is my MASTER TASK LIST. Hope this is detailed enough!

**Comment by teampsdkathryn** [comment]
I have completed a first draft as word and excel document. Please review and edit. I have sent to your respective emails as well. I am having trouble uploading into GitHub. Will try again

@lzim @dlounsbu @swapmush @teampsdkathryn @ staceypark

**Comment by teampsdkathryn** [comment]
[qual_tasklist v1_05_29_2019.xlsx](https://github.com/lzim/teampsd/files/3234351/qual_tasklist.v1_05_29_2019.xlsx)
[qual_tasklist v2_05_29_2019.docx](https://github.com/lzim/teampsd/files/3234352/qual_tasklist.v2_05_29_2019.docx)

I have completed a first draft of the qualitative work groups global task list as a word and excel document. It has been uploaded here. Please review and edit here or via email. Thank you, Kathryn

@lzim @dlounsbu @teampsdkathryn @swapmush @staceypark

**Comment by lzim** [comment]
Howdy!

@jamesmrollins @branscombj @dlkibbe @TomRust @holbrooa @saveth @swapmush @staceypark @teampsdkathryn @dlounsbu 

# Critical Path - Targeting 17 weeks 
Below is a **photo of our first draft of our critical path**, at least the end of today's work - thanks again to everyone for getting a task list with estimates done on time for today's prelaunch, project planning discussion!
- NOTE: Stacey and I worked on the HQ and Pubs/Conferences workstreams, which extended our critical path to R01/IIR launch substantially. We have marked off our schedules to protect a level of person hours that we believe we can sustain concurrent with our other responsibilities. 
- **Our target launch based on the critical paths reviewed today is likely October 1st, based on 17 weeks.** _As we discussed today, it may be more like 20 weeks_, but we will work to see if 17 might be realistic as we translate our network diagram to GANTT, and work to maximize our F2F time effectively.

# "Documentation"
- In the **quantitative workgroup** we further clarified our three types of MTL documentation based on level of specification and audience (it's in purple marker).

1. **DESCRIBE** > Data UI description, Sim UI Team Data Table "i" information > SEE guides (**learners**)
2. **DETAIL** > Data UI Column D and Data Notes > SAY guides (**facilitators**)
3. **DOCUMENT** > manuscripts > co-investigators (**scientists/experts**)

Andrew and Savet will break up their workgroup workflow into these levels, as other workgroups are dependent on getting #2, in particular, very soon!

# GANTT on OSF
- As I said in the MTL Launch meeting follow-up today, **we are working on a GANTT on OSF** that we will ask everyone to add to, edit and review as we further refine our plan.
- **Stacey plans to invite everyone tomorrow**, once the basic outline of our critical path details are added.

# F2F June 13-18
Our current schedule for our F2F June 13-18 (6 days of work) is to really emphasize these things, _**with as close to final level 1 DESCRIBE, and level 2 DETAIL (see above) "documentation: as possible.**_
- We would like to much better flesh this out based on what we identify **editing our GANTT by our meeting next Thursday 6/6**.
1. **Practice Data UI**
2. **Practice Sim UI**
3. **Practice Epicenter**

Thanks!

![prelaunch_critical_path](https://user-images.githubusercontent.com/7454688/58673967-9f761500-8302-11e9-81da-c130321cf3ad.jpeg)

**Comment by staceypark** [comment]
@lzim @dlounsbu @branscombj @holbrooa @dlkibbe @jamesmrollins @TomRust 
Below is the schedule drafted for the F2F!
The "prep + demo" items indicate 30 minutes of prep and 30 minutes of co-facilitation roleplay practice.
The "debrief + docu" items indicate time to discuss fidelity to session, document content for SAY files & resources.

Thurs   6/13 |   |   |   | Fri 6/14 |   |   |   | Sat 6/15 |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Task | Session | Min |   | Task | Session | Min |   | Task | Session | Min
Set-up/orient |   | 30 |   | Set-up/orient |   | 30 |   | Set up/orient |   | 30
mtl.how/data training |   | 60 |   | Prep + Demo | Session 5 MM | 60 |   | Demo | Session 6 PSY | 60
Break |   | 15 |   | Break |   | 15 |   | Break |   | 15
Prep + Demo | Session 2 | 60 |   | Prep + Demo | Session 5 PSY | 60 |   | Prep + Demo | Session 6 AGG | 60
Prep + Demo | Session 3 | 60 |   | Prep + Demo | Session 5 AGG | 60 |   | Prep + Demo | Session 6 SP | 60
Debrief + Docu | Session 2-3 | 15 |   | Lunch |   | 60 |   | Lunch |   | 60
Lunch |   | 60 |   | Prep + Demo | Session 5 SP | 60 |   | Debrief + Docu | Session 6 | 30
Prep + Demo | Session 4 | 60 |   | Debrief + Docu | Session 5 | 30 |   | Prep + Demo | Session 7 MM | 60
Debrief + Docu | Session 4 | 15 |   | Break |   | 15 |   | Break |   | 15
Break |   | 15 |   | Prep + Demo | Session 6 MM | 60 |   | Prep + Demo | Session 7 PSY | 60
mtl.how/facilitate training |   | 60 |   | Debrief |   | 30 |   | Debrief |   | 30
Debrief |   | 30 |   |   |   |   |   |   |   |  
Total |   | 8 |   |   |   | 8 |   |   |   | 8
  |   |   |   |   |   |   |   |   |   |  
Sun 6/16 |   |   |   | Mon 6/17 |   |   |   | Tues 6/18 |   |  
Task | Session | Min |   | Task | Session | Min |   | Task | Session | Min
Set up/orient |   | 30 |   | Set up/orient |   | 30 |   | Set up/orient |   | 30
Prep + Demo | Session 7 AGG | 60 |   | Prep + Demo | Session 8 SP | 60 |   | TeamPSD Meeting |   | 60
Break |   | 15 |   | Debrief + Docu | Session 8 | 30 |   | Break |   | 15
Prep + Demo | Session 7 SP | 60 |   | Break |   | 15 |   | Prep + Demo | Session 10 MM | 60
Debrief + Docu | Session 7 | 30 |   | Prep + Demo | Session 9 MM | 60 |   | Prep + Demo | Session 10 PSY | 60
Prep + Demo | Session 8 MM | 60 |   | Lunch |   | 60 |   | Lunch |   | 60
Lunch |   | 60 |   | Prep + Demo | Session 9 PSY | 60 |   | Prep + Demo | Session 10 AGG | 60
Prep + Demo | Session 8 PSY | 60 |   | Prep + Demo | Session 9 AGG | 60 |   | Break |   | 15
Prep + Demo | Session 8 AGG | 60 |   | Break |   | 15 |   | Prep + Demo | Session 10 SP | 60
Break |   | 15 |   | Prep + Demo | Session 9 SP | 60 |   | Debrief + Docu | Session 10 | 30
Debrief |   | 30 |   | Debrief + Docu | Session 9 | 30 |   | Debrief |   | 30
  |   |   |   |   |   |   |   |   |   |  
  |   | 8 |   |   |   | 8 |   |   |   | 8

**Comment by holbrooa** [comment]
Check out screenshot of quant workgroup's gantt chart. Andrew recommends Stacey get a copy of MS Project installed on her machine so that she can use software to manage this complex project with the right kind of tools. Notice how it tracks deliverables, shows different kinds of dependences, and shows resource overallocation.  Andrew and Savet had to use the unlicensed version, so we didn't have full functionality, but you can get the basic idea.
@lzim @staceypark 
https://osf.io/ktnz5

**Comment by lzim** [comment]
@holbrooa

Thanks we’ll take a look. I do want us to be able to properly use the GANTT over time, and we have considered Microsoft Project, but I wasn’t sure adding features, but making it more inaccessible for most was the way to go.

FYI: @staceypark

**Comment by holbrooa** [comment]
@lzim @staceypark 

Well, your and Stacey's opinions are more important here obviously, but I will say when I was using it on projects a lot, I never found inaccessibility to other people to be a problem. You can pass the left hand side, where the tasks and stuff are, as excel sheets and copy them into project directly, which makes the workers -> project manager workflow pretty easy. And all the useful chart stuff can be exported as pdfs or other things. Really, the primary use is to the people at the center of the web to manage dependencies and resource loads, so the last thing I would have wanted is other people to be able to come in and change my gantt directly. If they needed to change tasks or whatever, well, they have to go through me. And as I update the gantt, I can tell how that's affecting the critical path or other workflows and manage accordingly.  I recognize that we have very different project management styles though, so just my two cents.

I am curious - when you say "adding features" and "more inaccessible" what are you comparing to? I know Savet's used other project management software that I haven't seen, though she said it's pretty similar.

**Comment by lzim** [comment]
@holbrooa @saveth @staceypark @TomRust @branscombj @dlkibbe @dlounsbu @jamesmrollins 

> I am curious - when you say "adding features" and "more inaccessible" what are you comparing to? 

**_For our short-run need to think through our plan to manage the OMHSP TAS training and the F2F well,_** I'm comparing it to Excel > OSF for version control and delegating workgroups to update/revise their estimates today for tomorrow's meeting.

- However, once our short-run need to think this through, my preferred long range solution was that our Master Project Plan was integrated with existing workflows.
- It is NOT my preferred solution that we add a whole other process to our existing workgroups, nor is it my preferred solution that every update has to be provided to Stacey and I, rather than a more distributed, real-time approach.
- Moreover, I'm keeping my eye out with @jamesmrollins for solutions that work within our existing workflows and can also help to prioritize and queues of requests _from the field,_ not just from within our team.

**INTEGRATIONS**
I see. I had considered MS Project, as well as Ganttify, TeamGantt, OpenProject, (etc., etc.) and several other integrations that would work automatically with our existing GitHub and Lucid workflows.

Ideally once we have a working plan, our action item lists would be syncing with the master project plan to convey the percent completed toward key milestones. I do use this for very large milestones on GitHub here: https://github.com/lzim/teampsd/milestones

- However, I believe we need much smaller GitHub Issues and Smaller Milestones to use this better.

![milestones](https://user-images.githubusercontent.com/7454688/58967954-031c9a00-876a-11e9-8c94-38a6ae48f6b6.png)

MORE PROJECTS on GITHUB
- We also have the ability to use Kansan features of GitHub, which we set up at one point as one giant project, but have not kept current or broken down into smaller pieces. You can see that under https://github.com/lzim/teampsd/projects/1

![projects](https://user-images.githubusercontent.com/7454688/58968425-e59c0000-876a-11e9-938d-6dbce569913c.png)

**Comment by staceypark** [comment]
For the F2F schedule and facilitator practice, we made several decisions.
1. Lindsey will not be practicing during the F2F. This way we will have 2 rotating co-facilitator pairs between @branscombj @dlkibbe @TomRust @dlounsbu . Also, this allows Lindsey to float around and give feedback to co-facilitator pairs.

2. Our guides need to be rigorous enough that the 30 minutes prep for each session is enough. When we roll out with the r01 & iir teams, we will not have the time to be doing extra self-directed learning .

3. Every single facilitator needs to get 30+ hours of work in the sim. We need to be disciplined enough to stick to the schedule going through all sessions of MM, PSY, AGG, and SP (notice CC and Session 1, 11, & 12 was taken out as we do not have enough time). 

4. If we find a bug in the sim UI, we quickly alert @jamesmrollins and move on. If we find a bug in the data UI, we quickly alert @holbrooa and move on. We need to focus on the larger gestalt of the 12 session plan and not on testing/debugging.

5. Our goal for the F2F is rigorous practice around the theory of change.  Addressing the team need, walking through the guides, tailoring the intersession checklist, etc. all need to center around the theory of changes.

6. For the R01 grant, the control arm gets only the first 3 sessions of MTL. For the IIR grant, the control arm gets 12 sessions of "QI as usual". However, for the F2F, we are not going to be focusing on this for the F2F.

**_7. SAY reconciliation is our primary goal before the F2F._** Until now, we have been focusing on making our materials rigorous for our learners, but now we need to focus on the facilitators. For both our current and future facilitators (including the TAS), our materials need to be able to guide anyone in facilitating _Modeling to Learn_ focused on our theory of change.

**Comment by lzim** [comment]
@lzim @dlounsbu @branscombj @staceypark @TomRust @dlkibbe 

MTL Co-facilitator Fidelity Training Schedule (also on OSF):

[f2f_schedule.xlsx](https://github.com/lzim/teampsd/files/3272446/f2f_schedule.xlsx)

**Comment by lzim** [comment]
@lzim @dlounsbu @branscombj @staceypark @TomRust @dlkibbe

Hi Everyone,

We are going to give folks the #manuscriptmonday time back to focus on preparing for MTL Co-facilitator Fidelity Training or completing other work in advance of travel. Note that the OMHSP TAS are all in Washington D.C.

However, I will log in at 11AM Pacific to answer any questions you may have before the F2F.

@TomRust and @staceypark have specific tasks lists to prepare that I am working on with them.

@branscombj @dlounsbu and @dlkibbe As we indicated in the Team PSD Team Time report for this week how to prepare for the F2F we will be sending the four pre-work materials throughout the day today so that you have them. They are all relatively brief, and you can start now with #4, which we reviewed together on Friday. 😄 

1. REVIEW the orientation to the F2F training. (coming later today)
- The schedule is above, however, and on OSF.

2. READ the draft MTL facilitator orientation guide.
- This is going later today.

3.  REVIEW the 1st and 2nd pages of the facilitator cheatsheets.
- Headed your way later today.

4.  REVIEW the 2 intersession guides - available now here: https://github.com/lzim/teampsd/blob/master/mtl_facilitate_workgroup/checklists/inter_session_checklist.md

Thanks!

**Comment by staceypark** [comment]
@lzim @dlounsbu @branscombj @TomRust @dlkibbe
#3 facilitator cheatsheets still in progress. Heading your way later this evening 🛫 !
